integr8ly/application-monitoring-operator
# Intly 5184 blackbox self signed certs

Added logic to the reconcile loop in order to check for changes to the blackbox-exporter container config map. Should changes be detected, the prometheus pod will now be deleted in order to force a reload of the configmap.

JIRA: https://issues.redhat.com/browse/INTLY-5184

###
Verification
 - Clone this repo
 - Run the Operator 
 - Change the selfSignedCerts field of the application-monitoring CR and see that the prometheus pod reloads with the new configmap. This can be verified by viewing the availability of the blackbox metrics through Prometheus or Graphana changing with respect to the changed value in the application monitoring CR.
